src/test/java/jenkins/advancedqueue/jobinclusion/strategy/JobInclusionJobPropertyTest.java

```diff
@@ -0,0 +1,53 @@
+package jenkins.advancedqueue.jobinclusion.strategy;
+
+import static org.junit.Assert.*;
```

Comment:
Please replace with explicit imports rather than the wildcard import.  Wildcard import also includes the deprecated `assertThat` from org.junit.Assert and that is a distraction when using hamcrest assertions.
```suggestion
import static org.junit.Assert.*;
```

Reply:
Yes I have made the explicit imports for the functions used from org.junit.Assert.